django-stubs/core/management/commands/runserver.pyi

```diff
@@ -1,3 +1,7 @@
 from django.core.management.base import BaseCommand
 
-class Command(BaseCommand): ...
+class Command(BaseCommand):
+    default_addr: str
```

Comment:
Looking at other definitions, I believe this should be `default_addr: str = ...`.. Correct?

(Same applies to the other ones I've added here)

Reply:
Yes, this is a style thing mostly. Sorry, we don't have a way to check that automatically for now.

Reply:
No problem. Done :+1: